packages/@aws-cdk/aws-rds/lib/proxy.ts

```diff
@@ -524,3 +546,219 @@ function toConnectionPoolConfigurationInfo(
     sessionPinningFilters: props.sessionPinningFilters?.map(_ => _.filterName),
   };
 }
+
+/**
+ * The type of storage.
+ */
+export enum TargetRole {
+  /**
+   * Role for a proxy endpoint that can be used for both reading and writing.
+   */
+  READ_WRITE = 'READ_WRITE',
+
+  /**
+   * Role for a proxy endpoint that can only be used for reading.
+   */
+  READ_ONLY = 'READ_ONLY',
+}
+
+/**
+ * Options for a new DatabaseProxyEndpoint
+ */
+export interface DatabaseProxyEndpointOptions {
+  /**
+   * The identifier of the DB proxy endpoint.
+   *
+   *  @default - Generated by CloudFormation (recommended)
+   */
+  readonly dbProxyEndpointName: string;
```

Comment:
```suggestion
  readonly dbProxyEndpointName?: string;
```

Also change the `@default` to be "Generated by CDK"

Reply:
Good one